### ISSUE TITLE
Unknown production does not have a region specific emission factor for GB

### ISSUE THREAD

**Issue by OwenQWERTY** [electricitymap/electricitymap-contrib#4237]
No unknown production region specific emission factor for Great Britain, this leads to higher than representative carbon emissions for GB. The default is 700 assuming 100% fossil fuels.
![173194568-1e33f52b-b150-4e67-afee-a2c3fec95dc23](https://user-images.githubusercontent.com/67866804/173196575-3164f90e-ce51-4428-92cb-1baa249cfa3f.jpg)

**Comment by jarek** [comment]
Over the past 24 hours, "unknown" has contributed at most 9.21% of emissions (at 10 June 23:00 UTC) when it was contributing 1.84% of total production, so it's not a massive error.

Does anyone know of data about what this "unknown" production actually represents?

**Comment by jarek** [comment]
Just for housekeeping, this relates to https://github.com/electricitymap/electricitymap-contrib/issues/4236

**Comment by OwenQWERTY** [comment]
> Over the past 24 hours, "unknown" has contributed at most 9.21% of emissions (at 10 June 23:00 UTC) when it was contributing 1.84% of total production, so it's not a massive error.

Almost all of that will be around 2.33 times lower than reported, meaning the gCo2/kWh could be 4% less than reported on ElectricityMap. I am using an educated guess as to what the actual source of that energy would have for emissions by looking at ESO's definition of other. They say other (Unknown in this case) has an average gCo2/kWh of 300.

**Comment by jarek** [comment]
> Almost all of that will be around 2.33 times lower than reported, meaning the gCo2/kWh could be 4% less than reported on ElectricityMap.

Yes, however many if not all of carbon emission intensity values (like the 490 g CO2eq/kWh value we give to everything reported as "gas") are averages and approximations, where I wouldn't be surprised by 5-10% error. For example, the 490 g figure for gas comes from the IPCC 2014 report https://www.ipcc.ch/site/assets/uploads/2018/02/ipcc_wg3_ar5_annex-iii.pdf#page=7 where it is listed as min 410 g / median 490 g / max 650 g - that's -16% to +32% spread. So being 4% off isn't a big problem.

To get a figure with uncertainty range certain to be less than 5%, you would probably have to go through all plants in a region one by one and assess them in terms of exact fuel source (where does the gas or coal or biomass come from? what is its energy intensity? how was it extracted? how was it transported?), efficiency of the plant, any emissions control technology - and preferably do it in a peer-reviewed way because operators may have a motivation for stating their plants' emissions as low as possible. Then multiply the assessed carbon intensity by real-time contribution to the grid.

For reference, I went down the references list to find where carbonintensity.org.uk's intensity figures (like 394 g for closed-cycle gas turbine) listed in https://github.com/electricitymap/electricitymap-contrib/issues/4236#issuecomment-1152952419 came from and I got to _Measuring the progress and impacts of decarbonising British electricity_, Iain Staffell, 2017: https://www.sciencedirect.com/science/article/pii/S0301421516307017?via%3Dihub - this seems to use grid-wide values and so they wouldn't be taking per-plant efficiency times per-plant production into account either.

**Comment by OwenQWERTY** [comment]
Very good point, I see it is not important in the grand scheme of things. However I think it should at least have a lower gCo2/kWh ,700 is very harsh, I will try to see if I can find out what this 'Unknown' or 'Other' actually is.
